dev/local.env

```diff
@@ -20,6 +20,7 @@ export XMTPD_SETTLEMENT_CHAIN_NODE_REGISTRY_ADDRESS="0x8d69E9834f1e4b38443C63895
 export XMTPD_SETTLEMENT_CHAIN_RATE_REGISTRY_ADDRESS="0xE71ac6dE80392495eB52FB1dCa321f5dB8f51BAE"
 
 # Application Chain Options
+export XMTPD_APP_CHAIN_USE_DATABASE_CLIENT=false
```

Comment:
This is a placeholder flag that will likely change. For example, we will need the connection string for a read replica of the old node's database.